packages/yoroi-extension/features/step_definitions/connector-steps.js

```diff
@@ -280,9 +281,8 @@ Then(/^The wallet (.+) is connected to the website (.+)$/, async function (walle
     `Step: The wallet ${walletName} is connected to the website ${websiteUrl}`
   );
   await this.windowManager.switchTo(extensionTabName);
-  const connectedWebsitesAddress = `${this.getExtensionUrl()}#/connector/connected-websites`;
   // it should be reworked by using ui components when it is done
```

Comment:
@Nebyt  do you think we should remove this comment?!

Reply:
@AhmedIbrahim336 Not yet, after fully revamping the connector page we can remove

Reply:
Aha, got it! Thanks @Nebyt